egui/src/widgets/slider.rs

```diff
@@ -199,6 +202,28 @@ impl<'a> Slider<'a> {
         self
     }
 
+    /// Sets the minimal change of the value.
+    /// If `clamp_to_range` is enabled, `step` must be less than a maximum between the start and
+    /// the end of the range; otherwise, the step would be left unchanged.
+    /// Without `clamp_to_range` enabled, `step` can be any value.
+    ///
+    /// Default: `None`.
+    pub fn step_by(mut self, step: f64) -> Self {
+        let max = self.range.end().abs().max(self.range.start().abs());
+        let step = match self.clamp_to_range {
+            true if (step.abs() > max) => None,
+            _ => Some(step),
+        };
+        self.step = step;
+        self
+    }
+
+    /// Disable the `step_by` feature
+    pub fn disable_step(mut self) -> Self {
```

Comment:
No need for this - users should just not call `step_by` if they don't want a step.

Reply:
Removed.
But one can call steb_by(0.0) and this disables the feature. Turned out, it's convenient and I've used it in demo.